src/main/java/uk/nhs/hee/tis/trainee/sync/repository/ProgrammeRepository.java

```diff
@@ -38,6 +40,15 @@ public interface ProgrammeRepository extends MongoRepository<Programme, String>
   @Override
   Optional<Programme> findById(String id);
 
+  /**
+   * Find Programmes with the given owner.
+   *
+   * @param owner The owner to filter by.
+   * @return The found Programmes, empty if no results.
+   */
+  @Query("{ {'data.owner' : ?0}")
```

Comment:
Extra `{` that is likely to cause issues

Reply:
Yikes, thanks. 🔎